### ISSUE TITLE
Option to change namespace in REPL automatically when switching between files

### ISSUE THREAD

**Issue by avocade** [cursive-ide/cursive#2503]
I used Atom before and loved how it handled evaluating forms in different namespaces transparently in the REPL. Would be great to have a setting for this in Cursive as well.

Perhaps I haven't understood the correct way to configure this in Cursive, but it's quite a hassle to remember to not only `Load file` but also `Switch REPL to current namespace` every time we jump between files 😅

**Comment by cursive-ide** [comment]
You don't generally need to switch the REPL namespace, assuming that you're sending forms from your editor to the REPL rather than typing directly in the REPL itself. Cursive by default will evaluate forms sent from an editor in the namespace of that editor. If you want forms sent from the REPL to be evaluated in the active REPL namespace, you have to enable that explicitly at _Preferences | Languages & Frameworks | Clojure | REPL Options | Evaluate forms in REPL namespace_.

If this isn't what you were asking for and I've misunderstood something, let me know.

**Comment by eneroth** [comment]
Let's say I want to create a REPL command like this, for tapping values
![Screenshot 2021-02-19 at 11 40 23](https://user-images.githubusercontent.com/333974/108494022-57054480-72a7-11eb-9bc8-88f456a73c54.png)

How do I make this evaluate in the NS of the current file?

**Comment by avocade** [comment]
Yeah, I'm also using the `tap>` REPL command for convenience. But it's very inconvenient to have to remember to load the ns before using it every time 😂

(And just to be clear, `Cmd-Shift-P` works fine for me to evaluate the form in the current file without having to switch to that ns manually, as you said @cursive-ide. Great with some docs for that 👍)

**Comment by cursive-ide** [comment]
I see, yes, the REPL commands need an option for "current file namespace", I'll add that.

**Comment by avocade** [comment]
Great!
